### PR TITLE
Metrics for Compass director

### PR DESCRIPTION
**Pull request by kfurgol**
**Description**

Changes proposed in this pull request:

- Added dashboard and metrics for measuring tenants, applications and runtimes in compass Director
- Added dashboard and metrics for the postgres statistics

**Related issue(s)**
[#1235](https://github.com/kyma-incubator/compass/issues/1235)

### CODE REVIEW

File: resources/compass/charts/prometheus-postgres-exporter/values.yaml [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+    pg_compass_director_tenants_total:
+      query: "SELECT COUNT(ID) as quantity FROM business_tenant_mappings;"
+      master: true
+      metrics:
+        - quantity:
+            usage: "COUNTER"
+            description: "Quantity of total tenants in table"
+
+    pg_compass_director_tenants_active:
```
Comment (pkosiec) [inline resources/compass/charts/prometheus-postgres-exporter/values.yaml:305]: Please rename these metrics according to best practices: https://prometheus.io/docs/practices/naming/#metric-names
In this case, please put "_total" suffix.

File: resources/compass/charts/prometheus-postgres-exporter/values.yaml [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+      query: "SELECT COUNT(ID) as quantity FROM business_tenant_mappings WHERE status='Active';"
+      master: true
+      metrics:
+        - quantity:
+            usage: "COUNTER"
+            description: "Quantity of active tenants in table"
+
+    pg_compass_director_tenants_with_at_least_one_runtime_or_application:
```
Comment (pkosiec) [inline resources/compass/charts/prometheus-postgres-exporter/values.yaml:313]: Hmm this name is quite long. What do you think about `compass_director_used_tenants` or something like that?

File: resources/compass/charts/prometheus-postgres-exporter/values.yaml (hunk 1 above)
Comment (pkosiec) [reply resources/compass/charts/prometheus-postgres-exporter/values.yaml:305]: And I would recommend removing `pg_` suffix.

File: resources/compass/charts/prometheus-postgres-exporter/values.yaml [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+            usage: "COUNTER"
+            description: "Quantity of tenants which are assigned to at least one runtime or application"
+
+    pg_compass_director_applications:
+      query: "SELECT COUNT(ID) as quantity FROM applications;"
+      master: true
+      metrics:
+        - quantity:
+            usage: "COUNTER"
+            description: "Quantity of applications in table"
```
Comment (pkosiec) [inline resources/compass/charts/prometheus-postgres-exporter/values.yaml:327]: Replace "quantity" with "Total number"

File: resources/compass/charts/prometheus-postgres-exporter/values.yaml [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+    pg_compass_director_tenants_active:
+      query: "SELECT COUNT(ID) as quantity FROM business_tenant_mappings WHERE status='Active';"
+      master: true
+      metrics:
+        - quantity:
+            usage: "COUNTER"
+            description: "Quantity of active tenants in table"
+
+    pg_compass_director_tenants_with_at_least_one_runtime_or_application:
+      query: "SELECT COUNT (DISTINCT tenant_id) FROM (SELECT tenant_id FROM runtimes UNION ALL SELECT tenant_id FROM applications) a;"
```
Comment (pkosiec) [inline resources/compass/charts/prometheus-postgres-exporter/values.yaml:314]: what's `a` at the end of the query?

Comment (dbadura) [reply resources/compass/charts/prometheus-postgres-exporter/values.yaml:314]: `a` is the alias to the table in which we do `UNION`.

Comment (kfurgol) [reply resources/compass/charts/prometheus-postgres-exporter/values.yaml]: it's an output table of SQL subquery, I've renamed it to be more accurate

File: resources/compass/charts/director/templates/technical-dashboard-configmap.yaml [hunk 5]
```diff
@@ -1,18 +1,19 @@
-{{- if and .Values.global.metrics.enabled -}}
```
Comment (pkosiec) [inline resources/compass/charts/director/templates/technical-dashboard-configmap.yaml:1]: Let's not rename it, we will merge the dashboard eventually

**Pull request merged**